src/SCE/sce_engine.c

```diff
@@ -467,17 +488,19 @@ xccdf_test_result_type_t sce_engine_eval_rule(struct xccdf_policy *policy, const
 
 		if (fork_result == 0)
 		{
-		    // we won't read from the pipe, so close the reading fd
-		    close(pipefd[0]);
+		    // we won't read from the pipes, so close the reading fd
+		    close(stdout_pipefd[0]);
+		    close(stderr_pipefd[0]);
 
 		    // forward stdout and stderr to the opened pipe
-			dup2(pipefd[1], fileno(stdout));
-			dup2(pipefd[1], fileno(stderr));
+			dup2(stdout_pipefd[1], fileno(stdout));
```

Comment:
Looks like the previous comment is no longer valid.  "forward stdout to the opened pipe"